isolate_example/README.md

```diff
@@ -9,6 +9,31 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate with running the same calculation
+on a second isolate.When the main isolate is used, Flutter is unable to render new frames, and
+the `SmoothAnimationWidget`'s animation freezes.
+### `page_two.dart`
+
+Creates an isolate used to run an infinite loop that sums batches of 100M randomly generated
+numbers at a time. Users can start, terminate, pause, and resume the isolate, as well as modify
+how the calculation is performed.
+
+### `page_three.dart`
+
+Demonstrates how expensive it is to move large amounts of data between isolates and 
+better a alternative. This page creates an isolate that can add up a list of numbers and
```

Comment:
It looks like there's a cut-and-paste error in here or something: "...between isolates and better a alternative" doesn't parse grammatically.